src/containers/Row.js

```diff
@@ -0,0 +1,18 @@
+import { compose, withHandlers } from 'recompose'
+
+import Row from '../components/Row'
+
+const Cell = ({id, row}) => <td id={id}>{row[id]}</td>
+
```

Comment:
All UI code should be removed from the container.  We probably need to discus this, maybe it is OK sometimes?